### DISCUSSION TITLE
Support STOMP over WebSocket

### DISCUSSION THREAD

**Discussion by roie16** [nats-io/nats-server#5777] [Ideas]
Hi All,
Is there any plan to add STOMP support over WebSocket to enable app in JAVA to use nats as it's stomp broker relay(instead of RabbitMQ or ActiveMQ)?

**Comment by derekcollison** [comment]
We have a Java client you can use.

**Reply by roie16** [reply]
Hi @derekcollison, thank you for your quick reply, I am using the Java client, And it is working great as PUBSUB, but I want to use NATS as a STOMP broker relay, to leverage my Java app that is using websocket with a distributed broker, I was under the impression that NATS like RabbitMQ have [STOMP over Websocket support](https://www.rabbitmq.com/docs/stomp), But I can see now that it is not the case, So I don't think I will be able to use NATS for our project

https://stackoverflow.com/questions/78856083/spring-websocket-with-nats-as-stomp-broker-no-enum-constant-error/78856139#78856139

**Reply by derekcollison** [reply]
We have some customers that bridge between them but nothing OSS to do that today.

**Reply by roie16** [reply]
@derekcollison yes I understand that now, bummer, but all good :) , thank you for the help regardless
